### ISSUE TITLE
All sites other than Hitbox apparently fail to load

### ISSUE THREAD

**Issue by ghost** [JoselleAstrid/multisite-streams-page#7]
https://i.imgur.com/5OjweGM.png

Probably about a month ago I noticed this issue, where everything would just say "Waiting..." except for two Hitbox videos from users I still follow. I tested all three of the other sites and Twitch, Nico, and Cavetube are not working. At the moment it appears everything from Hitbox is working fine (except that only edge.sf has working thumbnails), but nothing else is.

**Comment by yoshifan** [comment]
Yeah, I had noticed this too - good to know it's not just me though (for that matter, nice to know someone else is trying this page). I suspect that Twitch finally closed the door on the old Twitch v3 API that this project uses; they said they would end support for it early this year.

I did a bit of work toward converting the Twitch code to use the New Twitch API, in the `twitch-overhaul` branch, but haven't made progress on that in a while. It's been difficult to find time for this project in general, but I'd like to continue it if possible. Hopefully I can at least fix Twitch by the end of the year.

**Comment by yoshifan** [comment]
Well, I've started to look at this again, so that's something I guess. Aiming to work on this steadily until I accomplish at least the following:

- [ ] Fix as much previous Twitch functionality as possible
- [ ] Fix as much previous Nicovideo functionality as possible
- [ ] Remove Cavetube
- [ ] Hitbox: Rename to Smashcast, and see if the thumbnail servers need updating

**Comment by JoselleAstrid** [comment]
All right, so after more thought, I've decided to drop this project after all. It's a lot to fix, the stream sites' APIs will continue to change over time (for better or worse), and I don't seem to have the bandwidth and motivation to keep up. Sorry about that.

I'm not sure if there are other projects similar to this, but hopefully other people are working on supporting alternative stream sites one way or another.
